packages/engine/src/interaction/behaviors/interactRaycast.ts

```diff
@@ -64,22 +63,23 @@ export const interactRaycast:Behavior = (entity: Entity, { interactive }:Interac
     // console.warn('!character.viewVector')
     return;
   }
+
   const raycaster = new Raycaster();
-  let object, rayOrigin, rayDirection;
+  let object, rayOrigin, rayDirection,rayCamera,rayMouse;
   // - added mouse raycaster
-  const rayCamera = Engine.camera.clone();
-  const rayMouse = mouseScreen.normalize();
+  rayCamera = Engine.camera.clone();
+  rayMouse = mouseScreen;
   raycaster.setFromCamera(rayMouse,rayCamera);
   let intersections = raycaster.intersectObjects(raycastList, true );
 
-  if (!intersections.length){
-  // TODO: rayOrigin, rayDirection
-  rayOrigin = clientPosition;
-  rayDirection = character.viewVector.clone().normalize().setY(0);
+//   if (!intersections.length){
+//   // TODO: rayOrigin, rayDirection
+//   rayOrigin = clientPosition;
+//   rayDirection = character.viewVector.clone().normalize().setY(0);
```

Comment:
make rayOrigin and rayDirection "const" here (remove them from "let" in line 68)